src/main/java/com/example/fooddelivery/model/Address.java

```diff
@@ -28,7 +28,7 @@
 
     @NotNull
     @Column
-    private Integer number;
+    private String number;
```

Comment:
why you changed the type of number?

Reply:
House numbers for example. We can have nr. 4a